src/main/java/org/prebid/server/proto/openrtb/ext/request/applogy/ExtImpApplogy.java

```diff
@@ -0,0 +1,13 @@
+package org.prebid.server.proto.openrtb.ext.request.applogy;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Value;
+
+@AllArgsConstructor(staticName = "of")
+@Value
+public class ExtImpApplogy {
+
+    @JsonProperty("token")
```

Comment:
This annotation is redundant since the json field name can be resolved from variable name. Please remove it,